src/libcollections/binary_heap.rs

```diff
@@ -760,3 +760,10 @@ impl<T: Ord> Extend<T> for BinaryHeap<T> {
         }
     }
 }
+
+#[unstable(feature = "extend_ref", reason = "recently added")]
```

Comment:
Due to the way stability works these instability annotations unfortunately won't be read and considered. In this case that's ok, though, but could you change these to `#[stable(feature = "extend_ref", since = "1.2.0")]`?